### PR TITLE
Reposition banner for iPhone X

### PR DESCRIPTION
**Pull request by christophercong**

### CODE REVIEW

File: Swift Demo App/iOS-SDK-Demo/ALDemoProgrammaticBannerViewController.swift [hunk 1]
```diff
@@ -22,17 +22,15 @@ class ALDemoProgrammaticBannerViewController: ALDemoBaseViewController, ALAdLoad
...
         
-        let toolbarHeight = navigationController?.toolbar.frame.height ?? 0
-        let frame = CGRect(x: 0, y: self.view.bounds.height - kBannerHeight - toolbarHeight, width: self.view.bounds.width, height: kBannerHeight)
-        adView = ALAdView(frame: frame, size: ALAdSize.sizeBanner(), sdk: ALSdk.shared()!)
-        
-        adView?.adLoadDelegate = self
-        adView?.adDisplayDelegate = self
-    
-        adView?.loadNextAd()
+        adView = ALAdView(sdk: ALSdk.shared()!, size: .sizeBanner())
```
Comment (thomasmso) [inline Swift Demo App/iOS-SDK-Demo/ALDemoProgrammaticBannerViewController.swift]: Is the banner actually centered horizontally?

Comment (christophercong) [reply Swift Demo App/iOS-SDK-Demo/ALDemoProgrammaticBannerViewController.swift]: Good question, I'll need to add support for that on Swift and Objective-C as well.

File: Swift Demo App/iOS-SDK-Demo/ALDemoProgrammaticBannerViewController.swift [hunk 2]
```diff
@@ -71,4 +77,30 @@ class ALDemoProgrammaticBannerViewController: ALDemoBaseViewController, ALAdLoad
     {
         self.log("Banner Clicked")
     }
+    
+    // MARK: Ad View Event Delegate
+    
+    func ad(_ ad: ALAd, didPresentFullscreenFor adView: ALAdView) {
```
Comment (thomasmso) [inline Swift Demo App/iOS-SDK-Demo/ALDemoProgrammaticBannerViewController.swift]: Could you keep styling consistent?

File: Swift Demo App/iOS-SDK-Demo/ALDemoProgrammaticMRecViewController.swift [hunk 3]
```diff
@@ -60,4 +69,30 @@ class ALDemoProgrammaticMRecViewController: ALDemoBaseViewController, ALAdLoadDe
...
+    
+    func ad(_ ad: ALAd, willLeaveApplicationFor adView: ALAdView) {
+        self.log("Banner will leave application")
+    }
+    
+    func ad(_ ad: ALAd, didReturnToApplicationFor adView: ALAdView) {
+        self.log("Banner did return to application")
+    }
+    
+    func ad(_ ad: ALAd, didFailToDisplayIn adView: ALAdView, withError code: ALAdViewDisplayErrorCode) {
```
Comment (thomasmso) [inline Swift Demo App/iOS-SDK-Demo/ALDemoProgrammaticMRecViewController.swift]: Could you keep styling consistent?

**Comment by thomasmso** [comment]
Feel free to merge

**Pull request merged**